include/bluetooth/conn.h

```diff
@@ -392,6 +392,8 @@ struct bt_conn_auth_cb {
 #if defined(CONFIG_BT_BREDR)
 	void (*pincode_entry)(struct bt_conn *conn, bool highsec);
 #endif
+	/* @brief indicate the authentication process is complete */
+	void (*pairing_complete)(struct bt_conn *conn, unsigned int status);
```

Comment:
Some issues I see with this:

 - The meaning and possible values for status will need to be documented. 

 - Are there defines/enum available for the possible values in some public header file? It seems this maps to the SMP status which is only available in the private `subsys/bluetooth/host/smp.h` header file currently.

 - Why `unsigned int` instead of `u8_t` as smp.c uses internally?

 - Since this maps to SMP errors it seems you don't plan to have this used for BR/EDR? Is it not useful there? Either way it needs to be documented that the application should not expect this callback for BR/EDR.

Reply:
Hi Johan, I added a enumeration to address your concern. But maybe we need to make the naming better. Any suggestions?  

As for BR/EDR, do you mean the suggestion you made last time about moving the callback to "struct bt_conn_cb"? I'm not familiar with BR/EDR part, can you confirm and advise further on this?